refinery/core/serializers.py

```diff
@@ -47,6 +49,16 @@ def get_owner(self, analysis):
     def get_data_set_uuid(self, analysis):
         return analysis.data_set.uuid
 
+    def get_creation_date(self, analysis):
+        return analysis.time_start
+
+    def get_workflow_json(self, analysis):
```

Comment:
This should be added to the workflow serializer and the client should make the appropriate call.